sdk/modelsrepository/azure-iot-modelsrepository/src/main/java/com/azure/iot/modelsrepository/implementation/StandardStrings.java

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.iot.modelsrepository.implementation;
+
+public class StandardStrings {
```

Comment:
Feels like the name should be `StatusStrings`

Reply:
Sure, changing.